extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -2249,14 +2249,16 @@ public static function onGetUserPermissionsErrors( Title $title, User $user, str
 			switch ( $action ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "case default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)